src/views/Teste.vue

```diff
@@ -0,0 +1,16 @@
+<script lang="ts" setup>
```

Comment:
You dont need this component. Delete it please.

Reply:
done